src/containers/SearchPageContainer.tsx

```
@@ -0,0 +1,95 @@
+'use client';
+
+import { useCallback, useEffect, useRef, useState } from "react";
```

ESLint

[error] 'useEffect' is defined but never used.

(@typescript-eslint/no-unused-vars)